.github/PRIVACY_POLICY.md

```diff
@@ -2,18 +2,18 @@
 
 ## 1. Information
 ### 1a. What information do we collect from the people that visit our website?
-When using Memerator owned products, there is some information required in order to use the software.
+When using Memerator, there is some information required in order to use the software.
 The information includes a user uploaded image, and top & bottom text to include on the meme.
 This information is for the meme generation only, and will not be stored on our server, nor will we be able to see it.
 ### 1b. Why do we collect information?
 If you do not enter this information, the app is pointless because it makes no meme.
 
 ## 2. Cookies
 ### 2a. Do we use 'cookies'?
-We use cookies for uses such as for tracking purposes.
+We use cookies for analytic tracking purposes.
 We do not store any cookies on your computer except for third-party cookies.
 ### 2b. Why do we use 'cookies'?
-Without allowng third-party services use cookies, we can't use them.
+We use cookies for analytic tracking purposes.
```

Comment:
For this part, it is pointless having the same as 2a. So we can still change it but keep it different than the previous part.